### PR TITLE
Playground: Allow hide `input` and `output`

### PR DESCRIPTION
**Pull request by fisker**
## Description

When `show doc`, I think user mean to focus checking `Doc`, don't care "output" that much.

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by thorn0** [comment]
It's often important to see what output the doc generates. The user changes the input and immediately can see how it affects both the doc and the output. That's how I often use it. Sometimes small changes in the doc, trigger not very expected changes in the output, and it's better to see that. If you want to hide the output, add an independent checkbox for that.

**Comment by thorn0** [comment]
Is it for mobile too?

**Comment by fisker** [comment]
I just want more space for the panel I'm focusing, yes better on mobile too .

File: website/playground/Playground.js [hunk 1]
```diff
@@ -242,6 +242,16 @@ class Playground extends React.Component {
...
+                        <Checkbox
+                          label="show output"
+                          checked={editorState.showOutput}
+                          onChange={editorState.toggleOutput}
+                        />
+                        <Checkbox
+                          label="show input"
+                          checked={editorState.showInput}
+                          onChange={editorState.toggleInput}
+                        />
```
Comment (thorn0) [inline website/playground/Playground.js:254]: Maybe let's change the order of the checkboxes to match the order of the panels?

**Comment by fisker** [comment]
I was thinking that these two buttons won't be used as 
 much as others.

File: website/playground/Playground.js (hunk 1 above)
Comment (fisker) [reply website/playground/Playground.js:254]: I was thinking that these two buttons won't be used as
much as others.

Comment (thorn0) [reply website/playground/Playground.js:254]: That's true, but the order looks strange. Let's change it.

Comment (fisker) [reply website/playground/Playground.js:254]: I'm happen to be an expert to  cure OCD, want help? I can give you discount. 😄

Comment (fisker) [reply website/playground/Playground.js:254]: Can you edit it? I'm not near my laptop.

**Pull request merged**